tools/release_provenance/models.py

```diff
@@ -26,7 +26,7 @@ class ComponentBuild(Base):
     spec = Column(String, nullable=False)
     install_path = Column(String, nullable=False, unique=True)
     created_at = Column(DateTime, nullable=False)
-    release_url = Column(Text, nullable=False, unique=True)
+    release_url = Column(Text, nullable=False)
```

Comment:
This was changed to non-unique just in case? Seemed an unnecessary constraint? Or one we didn't care too much about enforcing?

Reply:
Release URLs for components (mom5, cice5,...) will point to the same overall model release (eg. https://github.com/ACCESS-NRI/ACCESS-OM2/releases/tag/2024.03.0) which will be non-unique, so I removed that constraint.